doc/news/_preparation_next_release.md

```diff
@@ -183,6 +183,7 @@ you up to date with the multi-language support provided by Elektra.
   location data, providing additional features, such as clickable links to the error source. _(René Schwaiger)_
 
 - Added mores specification examples. _(Michael Zronek)_
+
```

Comment:
Please describe which examples you added (and link to them). "Added mores specification examples." is not very helpful.